jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/codescan/sessionconfig/JavaScriptCodeScanSessionConfig.kt

```diff
@@ -61,6 +69,9 @@ internal class JavaScriptCodeScanSessionConfig(
 
     override fun getImportedFiles(file: VirtualFile, includedSourceFiles: Set<String>): List<String> {
         val importedFiles = mutableListOf<String>()
+//        if (this.selectedFile.programmingLanguage().toTelemetryType() === CodewhispererLanguage.Javascript){
+//            val imports = parseImports(file)
+//        }
```

Comment:
do we need this

Reply:
Oops,
I will remove these commented lines.